yt/frontends/boxlib/data_structures.py

```diff
@@ -621,7 +621,7 @@ class BoxlibDataset(Dataset):
     _output_prefix = None
 
     # THIS SHOULD BE FIXED:
```

Comment:
@matthewturk can I have your review on this comment you wrote ? I don't know what was there to fix (default value, type, anything else ?) and if that comment should be cleaned up or updated.